src/System.Management.Automation/engine/Modules/ModuleCmdletBase.cs

```diff
@@ -1837,23 +1837,16 @@ private IEnumerable<PSModuleInfo> CreateFakeModuleObject(IEnumerable<ModuleSpeci
             else if ((requiredProcessorArchitecture != ProcessorArchitecture.None) &&
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-static-analysis`
Verify Markdown Links.Verify links in /home/vsts/work/1/s/docs/learning-powershell/powershell-beginners-guide.md.https://www.computerperformance.co.uk/powershell/index.htm should work
```powershell
retry of URL failed with error: Resource temporarily unavailable
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 117
117:                                     throw "retry of URL failed with error: $($_.Exception.Message)"

```